flocker/control/httpapi.py

```diff
@@ -790,7 +802,16 @@ def list_current_nodes(self):
                 self.cluster_state_service.as_deployment().nodes]
 
     @app.route("/configuration/_compose", methods=['POST'])
-    @private_api
+    @user_documentation(
+        u"""
+        Private API endpoint used by flocker-deploy.
+
+        Please do not use it as it may be removed in the near future.
```

Comment:
I think I'd still rather use the `@private_api` decorator, but if you stick to this at least update docstring appropriately.